### ISSUE TITLE
Bug in lumino widgets

### ISSUE THREAD

**Issue by umerhasan17** [jupyterlab/jupyterlab#7954]
## Description

I have been following along the random astronomy picture of the day tutorial and came across this problem when using the package @lumino/widgets instead of @phosphor/widgets. The rest of it is identical. Is it something I am doing wrong or is it a bug within this newly created package I gather? The problem is in line 26.

## Code for reproducing 

```
import {
  JupyterFrontEnd, JupyterFrontEndPlugin
} from '@jupyterlab/application';

import {
  ICommandPalette, MainAreaWidget
} from '@jupyterlab/apputils';

import {
  Widget
} from '@lumino/widgets';


/** 
 * Initialization data for the weather_ext extension.
 */
const extension: JupyterFrontEndPlugin<void> = {
  id: 'weather_ext',
  autoStart: true,
  requires: [ICommandPalette],
  activate: (app: JupyterFrontEnd, palette: ICommandPalette) => {
    console.log('JupyterLab extension jupyterlab_apod is activated!');
  
    // Create a blank content widget inside of a MainAreaWidget
    const content = new Widget();
    const widget = new MainAreaWidget({content});
    widget.id = 'apod-jupyterlab';
    widget.title.label = 'Astronomy Picture';
    widget.title.closable = true;
  
    // Add an application command
    const command: string = 'apod:open';
    app.commands.addCommand(command, {
      label: 'Random Astronomy Picture',
      execute: () => {
        if (!widget.isAttached) {
          // Attach the widget to the main work area if it's not there
          app.shell.add(widget, 'main');
        }
        // Activate the widget
        app.shell.activateById(widget.id);
      }
    });
  
    // Add the command to the palette.
    palette.addItem({command, category: 'Tutorial'});
  }
  
};


export default extension;
```

## Expected behavior

Same as phosphor widget

## Context

- Ubuntu 18.04.4 LTS

<details><summary>Command Line Output</summary>
<pre>
      Types of property 'icon' are incompatible.
        Type 'string | IRenderer' is not assignable to type 'string'.
          Type 'IRenderer' is not assignable to type 'string'.

26     const widget = new MainAreaWidget({content});
                                          ~~~~~~~

  node_modules/@jupyterlab/apputils/lib/mainareawidget.d.ts:83:9
    83         content: T;
               ~~~~~~~
    The expected type comes from property 'content' which is declared here on type 'IOptions<Widget>'

Found 2 errors.

</pre>
</details>

**Comment by jasongrout** [comment]
We just released JupyterLab 2.0. Are you using the 2.0 version of the tutorial with the JupyterLab 2.0 release?

**Comment by umerhasan17** [comment]
I upgraded to 2.0 and it builds. Thanks @jasongrout
